src/app/data/calculations.ts

```diff
@@ -13,18 +14,30 @@ export interface Calculation<T = any> {
 export const calculations: Calculations = {
     'Maximum': {
         type: ScoutMetricType.NUMBER,
-        invoke: (...metrics: Array<Metric>) => metrics[0]
+        invoke: (...metrics: Array<Metric>) => reduce(metrics, function (acc: Metric, metric) {
```

Comment:
Not too big of a deal, but arrow functions are beautiful

```suggestion
        invoke: (...metrics: Array<Metric>) => reduce(metrics, (acc: Metric, metric) => {
```